client/types.go

```diff
@@ -427,7 +427,11 @@ type Signup struct {
 }
 
 type Tenant struct {
-	Signup Signup `json:"signup"`
+	Signup              Signup  `json:"signup"`
+	FromEmail           *string `json:"fromEmail,omitempty"`
+	FinanceSupportEmail *string `json:"financeSupportEmail,omitempty"`
+	SiteAccessCode      *string `json:"siteAccessCode,omitempty"`
+	StateEvent          *string `json:"stateEvent,omitempty"`
```

Comment:
Consider `StateEvent` to be an enum, not an open string type

Reply:
we decided meanwhile not implement StateEvent, so deleted. Thank you for comment